File: java-lhdiff-project/datasets/Aakanksha/File08_Old.java
// File08_Old.java
public class File08 {
    private int counter = 0;

    public void increment() {
        counter++;
        System.out.println("Counter: " + counter); // This line
    }

    public void resetAndLog(int initialValue) {
        this.counter = initialValue;
        String logMessage = "Counter reset to " + initialValue + ".";
        System.out.println(logMessage);
    }
}

File: python-lhdiff-project/datasets/Aakanksha/File08_Old.py
# File08_Old.py

class File08:
    def __init__(self):
        self.counter = 0

    def increment(self):
        self.counter += 1
        print("Counter:", self.counter)  # This line

    def reset_and_log(self, initial_value):
        self.counter = initial_value
        log_message = f"Counter reset to {initial_value}."
        print(log_message)